src/transfers/MultiAppMooseOkapiTransfer.C

```diff
@@ -169,50 +173,128 @@ MultiAppMooseOkapiTransfer::execute()
+          ErrorHandling::openmc_tally_get_filters(err_get, "MultiAppMooseOkapiTransfer");
+
+          if (num_filter_indices != 2)
+            mooseError(
+                "We expect there to be exactly two filters, one a cell filter and the other an "
+                "expansion filter. Check the tallies XML file to verify the existence of both for "
+                "the requested tally");
+          std::map<int32_t, int32_t> index_to_id;
+          for (decltype(n_cells) i = 0; i < n_cells; ++i)
+          {
+            int32_t id;
+            err_get = openmc_cell_get_id(i + 1, &id);
+            ErrorHandling::openmc_cell_get_id(err_get, "MultiAppMooseOkapiTransfer");
+            index_to_id[i + 1] = id;
+          }
+
+          // Important variables
+          int32_t order;
+          char type[20];
+          int32_t * cell_indices = nullptr;
+          int32_t num_cells_in_filter;
+          bool cell_id_found = false;
+          int32_t stride_integer = 0;
+          int32_t cell_filter_index;
+
+          std::string cell_filter_name = "cell";
+          bool cell_filter_found = false;
+          for (int32_t i = 0; i <= 1; ++i)
+          {
+            err_get = openmc_filter_get_type(filter_indices[i], type);
+            ErrorHandling::openmc_filter_get_type(err_get, "MultiAppMooseOkapiTransfer");
+            if (cell_filter_name.compare(type))
+              continue;
+            else
+            {
+              cell_filter_found = true;
+              cell_filter_index = i;
+              break;
+            }
+          }
+          if (!cell_filter_found)
+            mooseError("No cell filter specified. Check the tallies XML input file.");
+          else
+          {
+            err_get = openmc_cell_filter_get_bins(
+                filter_indices[cell_filter_index], &cell_indices, &num_cells_in_filter);
+            ErrorHandling::openmc_cell_filter_get_bins(err_get, "MultiAppMooseOkapiTransfer");
+
+            for (decltype(num_cells_in_filter) i = 0; i < num_cells_in_filter; ++i)
+            {
+              if (cell_indices[i] == _cell_index)
+              {
+                cell_id_found = true;
+                stride_integer = i;
+                break;
+              }
+            }
+            if (!cell_id_found)
+              mooseError("Requested cell_id not in the passed tally. Check that the cell filter in "
+                         "the tallies XML file contains the ID you're requesting");
+          }
+          int32_t expansion_filter_index = 1 - cell_filter_index;
+          err_get = openmc_filter_get_type(filter_indices[expansion_filter_index], type);
+          ErrorHandling::openmc_filter_get_type(err_get, "MultiAppMooseOkapiTransfer");
+          getOrderAndCheckExpansionType(type, filter_indices[expansion_filter_index], order);
+
+          Real multiplier = _geometry_type == "cylindrical" ? 2. : 1.;
```

Comment:
Is this here to correct for different normalizations between OpenMC and the MOOSE FE App, or something else? It's minor, but if this isn't going to change we could make `_multiplier` a member variable that's set in the constructor based on the value of `_geometry_type`.